js/scripts.js

```diff
@@ -0,0 +1,22 @@
+
+ var a = 4;
+ var b = 4;
+ value = (a * a) + (2 * a * b) - (b * b)
+
+if ( value > 0 ) {
+	 console.log('Wynik jest dodatni');
+
+} else  {
+	 console.log('Wynik jest ujemny');
+
+}
+
+//did value is equal zero
+
+if ( value = 0 ) {
+	console.log('wynik jest równyt zero');
+
+} else {
+	console.log('wynik nie jest równay zero');
+}
+
```

Comment:
masz dwa oddzielne ify + elsy: zrób to przez jedną kombinację `if` + `else if` + `else`